package.json

```diff
@@ -1,15 +1,25 @@
 {
-  "name": "fishsim",
-  "version": "4.1.0",
+  "name": "rmkfish",
+  "version": "0.1.0",
```

Comment:
We should probably stick with `fishsim`, and bump either the major version (to 5.0.0) or the minor (to 4.2.0)?